### PR TITLE
[VEN-2394] Deploy Prime to opBNB testnet

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: deployments/opbnbtestnet/DefaultProxyAdmin.json
Comment (GitGuru7) [inline deployments/opbnbtestnet/DefaultProxyAdmin.json:1]: We should maintain single `DefaultProxyAdmin` per network, For the new deployments on opBNB `DefaultProxyAdmin` of [oracle](https://github.com/VenusProtocol/oracle/blob/develop/deployments/opbnbtestnet/DefaultProxyAdmin.json) is being used. Same goes with every network.

File: deployments/opbnbtestnet/PrimeLiquidityProvider_Proxy.json
Comment (GitGuru7) [inline deployments/opbnbtestnet/PrimeLiquidityProvider_Proxy.json:1]: Contracts are not verified.

File: deploy/008-deploy-prime.ts [hunk 1]
```diff
@@ -28,22 +29,26 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
...
     ethereum: 0,
+    opbnbtestnet: 0,
   };
 
   const blocksPerYear: Config = {
     bsctestnet: 10_512_000, // 3 sec per block
     sepolia: 2_628_000, // 12 sec per block
     bscmainnet: 10_512_000,
     ethereum: 2_628_000,
+    opbnbtestnet: 0,
```
Comment (chechu) [inline deploy/008-deploy-prime.ts:40]: 31536000? 1 block per second

File: deploy/008-deploy-prime.ts [hunk 2]
```diff
@@ -109,13 +109,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   });
 
   const prime = await ethers.getContract("Prime");
-  await prime.initializeV2(ADDRESSES[networkName].poolRegistry);
+  // await prime.initializeV2(ADDRESSES[networkName].poolRegistry);
```
Comment (chechu) [inline deploy/008-deploy-prime.ts:112]: To be removed and added to the VIP

**Pull request closed without merging**